body.tex

```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
 agile way, while also ensuring that scope, cost and schedule are
 carefully controlled.
 
+
+\subsection{Defining the Schedule with Milestones}
+
+Rubin Milestones are defined as Jira issues of type ``Milestone''. As indicated above, the Director (or their designate) defines the L1 milestones, the Associate Directors (ADs) define their departments' L2 milestones, and the Team Leaders and ADs define the L3 milestones for their teams.
+
+The following fields must be filled out when defining a new milestone:
+\begin{itemize} 
+\item 
+  The \texttt{Milestone Level}, ``1,'' ``2,'' or ``3;''
+\item
+  The \texttt{RO Milestone ID}. 
+  This is a string like ``L2-DI-0003'' which indicates the level of the milestone, the department (for level 2 milestones) or team (for level 3 milestones) that owns it, and an ID number that serves to make the ID unique wbut which otherwise carries no meaning. 
+  The list of two-character department and team string identifiers can be found in the Appendix (``DI'' in the example stands for ``Directorate''). 
+  The ID number may get edited by the Director's Office to ensure that it is unique: the Milestone IDs are only ever used in linking together the milestones for visualization in Smartsheet. 
+  Good practice here is to look at the milestone list in Smartsheet and choose an ID number that leads to a milestone ID that has not already been taken, and which roughly fits your milestone into the time-ordered list.  
+\item
+  The \texttt{Summary} field should contain a sentence outlining either what needs to be done, or what will be delivered for the Milestone to be reached. 
+  Examples include \textit{``Deliver Data Preview 0.1 (DP0.1)''} (an L1 milestone) and \textit{``DP0.1 Data Release: science-ready catalogs released from the IDF''} (an L2 milestone that belongs to it).
+  % PJM: Should we homogenize the style here, and make Milestone summaries EITHER directives OR deliverables? Discussed with Amanda: leave this up to the teams' preferences.
+\item  
+  The \texttt{POP Activity Description} text should contain one or maybe two sentences summarizing the activity needed to complete the milestone. 
```

Comment:
why specify "POP" here if we are doing the same thing for all milestones (even non-Pop milestones)?   I think that's confusing.

Reply:
```suggestion
  The \texttt{Activity Description} text should contain one or maybe two sentences summarizing the activity needed to complete the milestone. 
```

Reply:
We can drop the "POP" from the text here, but in Jira the name of the field is literally "POP Activity Description" for all issues of type "milestone", even if they are Level 2 or Level 3. We should change that to make the field simply "Activity Description" I think. Paging @cathypetry !